### ISSUE TITLE
Add `sum` to `_sparse_py_operators`

### ISSUE THREAD

**Issue by aerubanov** [aesara-devs/aesara#522]
When working on https://github.com/pymc-devs/pymc3/pull/4596, I needed the sum() method for sparse matrices for doing things like these:
```python
import numpy as np, aesara as as

 W = np.array(
            [[0.0, 1.0, 1.0, 0.0], [1.0, 0.0, 0.0, 1.0], [1.0, 0.0, 0.0, 1.0], [0.0, 1.0, 1.0, 0.0]]
        )
W = as.tensor.as_tensor_variable(W)
W = as.sparse.csr_from_dense(W)
D = W.sum(axis=0)
```
and as a result, D should be dense vector ```[2.0, 2.0,  2.0, 2.0]```.

**Comment by twiecki** [comment]
@aerubanov Can you add more complete example code that imports modules and creates a sparse matrix?

**Comment by brandonwillard** [comment]
There is already support for sums on sparse matrices; see [here](https://github.com/aesara-devs/aesara/blob/95deb922b2a3a965e1d09308b6edde54358a8b67/aesara/sparse/basic.py#L1808).  If you're having specific issues with this functionality, you can open another issue or update this one with a minimum working example; otherwise, I'll assume this particular issue is only about adding a `sum` _method_.

**Comment by aerubanov** [comment]
@brandonwillard Thank you, looks like it is exactly what I need! I just couldn't find it in the documentation. So I close the issue.

**Comment by brandonwillard** [comment]
We should still have a `sum` method, so I'm reopening it for that.

**Comment by aerubanov** [comment]
This feature was implemented in #745, so I close issue.
